plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableHandle.java

```diff
@@ -55,6 +55,9 @@ public class IcebergTableHandle
     // Filter guaranteed to be enforced by Iceberg connector
     private final TupleDomain<IcebergColumnHandle> enforcedPredicate;
 
+    // Columns that are explicitly present in predicates in the query
```

Comment:
nit: `Columns that are present in {@link Constraint#predicate()} applied on the table scan`